news-bundle/src/EventListener/DataContainer/PageListener.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\NewsBundle\EventListener\DataContainer;
+
+use Contao\BackendUser;
+use Contao\CoreBundle\ServiceAnnotation\Callback;
+use Contao\CoreBundle\ServiceAnnotation\Hook;
+use Contao\DataContainer;
+use Contao\NewsBundle\Controller\Page\NewsFeedController;
+use Doctrine\DBAL\Connection;
+use Symfony\Component\Security\Core\Security;
+
+class PageListener
+{
+    public function __construct(private Connection $connection, private Security $security)
+    {
+    }
+
+    /**
+     * @Callback(table="tl_page", target="config.onload")
+     */
```

Comment:
All of these should be PHP 8 attributes.

Reply:
Fixed in d8ddd2965